### DISCUSSION TITLE
q-select pwa difficult to close dialog

### DISCUSSION THREAD

**Discussion by franfrLor** [quasarframework/quasar#10544] [CLI - PWA mode]
I have a PWA and put it on the home screen under IOS.
Using this shortcut, the q-select lists open in the form of a dialog and to get out of it I have to make 5 clicks outside the list.
Going through Safari, everything works fine a single click closes the dialog window.
How can I solve this problem?

**Comment by twis4t** [comment]
I have the same problem.
In the spa mode work normally.
